ios/App/App/AppDelegate.swift

```diff
@@ -0,0 +1,74 @@
+import UIKit
+import Capacitor
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  var window: UIWindow?
+
+
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 143 characters](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6209007)